### ISSUE TITLE
Importing __version__ in setup prevents dependency resolution

### ISSUE THREAD

**Issue by daf** [chronitis/ipyrmd#4]
Trying to install ipyrmd in a fresh environment as part of a requirements.txt file results in it trying to import `__version__`, which is located in a manner that tries to import all its dependencies, which aren't yet installed:

```
Collecting nbformat (from -r requirements.txt (line 2))
  Downloading nbformat-4.3.0-py2.py3-none-any.whl (154kB)
...
Collecting ipyrmd (from -r requirements.txt (line 9))
  Downloading ipyrmd-0.4.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-3qfxr3lw/ipyrmd/setup.py", line 4, in <module>
        from ipyrmd import __version__
      File "/tmp/pip-build-3qfxr3lw/ipyrmd/ipyrmd/__init__.py", line 4, in <module>
        from .ipyrmd import ipynb_to_rmd, rmd_to_ipynb, ipynb_to_spin, spin_to_ipynb
      File "/tmp/pip-build-3qfxr3lw/ipyrmd/ipyrmd/ipyrmd.py", line 17, in <module>
        import nbformat
    ModuleNotFoundError: No module named 'nbformat'
```

This is pip 9.0.1.

Perhaps `__version__` can be pulled from a different location, or a `try`/`except ImportError` can be wrapped around the other imports in `__init__.py`?

**Comment by chronitis** [comment]
Should be fixed in 0.4.3, sorry for the delay.

**Comment by daf** [comment]
Thanks!
